### PR TITLE
OTLP Logs Intake Endpoint

### PR DESCRIPTION
**Pull request by brett0000FF**
### What does this PR do? What is the motivation?

- New topic for Datadog OTLP Logs Intake Endpoint
- Placed under Interoperability menu heading for now.

### Merge instructions

Merge readiness:
- [ ] Ready for merge

Merge queue is enabled in this repo. To have it automatically merged after it receives the required reviews, create the PR (from a branch that follows the `<yourname>/description` naming convention) and then add the following PR comment:

```
/merge
```

### CODE REVIEW

File: content/en/opentelemetry/otlp_logs.md [hunk 1]
```diff
@@ -0,0 +1,122 @@
...
+
+{{< site-region region="ap1,gov" >}}
+<div class="alert alert-warning">Datadog OTLP logs intake endpoint is not supported for your selected <a href="/getting_started/site">Datadog site</a> ({{< region-param key="dd_site_name" >}}).</div>
+{{< /site-region >}}
+
+## Overview
+
+Datadog's OpenTelemetry Protocol (OTLP) logs intake API endpoint allows you to send logs directly to Datadog. With this feature, you don't need to run the [Datadog Agent][2] or [OpenTelemetry Collector + Datadog Exporter][1].
+
+You might prefer this option if you're looking for a straightforward setup and want to send logs directly to Datadog without using the Datadog Agent or OpenTelemetry Collector.
```
Comment (estherk15) [inline content/en/opentelemetry/otlp_logs.md:24]:
```suggestion
Choose this option for a straightforward setup to send logs directly to Datadog without using the Datadog Agent or OpenTelemetry Collector.
```

File: content/en/opentelemetry/otlp_logs.md [hunk 2]
```diff
@@ -0,0 +1,122 @@
...
+
+```shell
+export OTEL_EXPORTER_OTLP_LOGS_PROTOCOL="http/protobuf"
+export OTEL_EXPORTER_OTLP_LOGS_ENDPOINT="{{< region-param key="otlp_logs_endpoint" >}}"
+export OTEL_EXPORTER_OTLP_LOGS_HEADERS="dd-protocol=otlp,dd-api-key=${DD_API_KEY}"
+```
+
+#### Manual instrumentation
+
+If you are using manual instrumentation with OpenTelemetry SDKs, configure the OTLP HTTP Protobuf exporter programmatically using the examples below.
```
Comment (estherk15) [inline content/en/opentelemetry/otlp_logs.md:44]:
```suggestion
If you are using manual instrumentation with OpenTelemetry SDKs, configure the OTLP HTTP Protobuf exporter programmatically using the following examples.
```

**Review by estherk15 — APPROVED** [review]
Looks great!

**Review by csantosdd — APPROVED** [review]

**Pull request merged**